File: C++/Coterminal Angles.cpp
#include <cmath>
#include <iostream>
using namespace std;

int main()
{
	cout <<"State an angle.\n";
	__int64 o;
	cin >> o;
	o=o%360;
	cout <<"Coterminal angles include "<<o<<" and " <<o-copysignl(360,o)<<".\n";
}

File: Visual Basic .NET/Coterminal Angles.vb
Module Module1
    Sub Main()
        Console.WriteLine("Please state your angle.")
        Dim o As Int64 = Console.ReadLine() Mod 360
        Console.WriteLine("The coterminal angles include {0} and {1}.", o, o - Math.Sign(o) * 360)
    End Sub
End Module